compiler-rt/lib/hwasan/CMakeLists.txt

```diff
@@ -15,16 +15,28 @@ set(HWASAN_RTL_SOURCES
   hwasan_memintrinsics.cpp
   hwasan_poisoning.cpp
   hwasan_report.cpp
-  hwasan_setjmp_aarch64.S
-  hwasan_setjmp_riscv64.S
-  hwasan_setjmp_x86_64.S
-  hwasan_tag_mismatch_aarch64.S
-  hwasan_tag_mismatch_riscv64.S
   hwasan_thread.cpp
   hwasan_thread_list.cpp
   hwasan_type_test.cpp
   )
 
+foreach(arch ${HWASAN_SUPPORTED_ARCH})
```

Comment:
For my education, why do we need to `foreach` over `HWASAN_SUPPORTED_ARCH` (not `ARCHS`)?

Reply:
Nevermind.